packages/ui/v2/core/Meta.tsx

```diff
@@ -1,18 +1,18 @@
 import Head from "next/head";
 import React, { createContext, useContext, useState, useEffect } from "react";
 
-import { useLocale } from "@calcom/lib/hooks/useLocale";
-
 type MetaType = {
   title: string;
```

Comment:
NIT: @joeauyeung seeing this file the first time, can you explain me what this does? And why do we need CTA in there. It seems like this is only handling `<meta>` data

Reply:
@zomars jump in if I'm missing anything. The `Meta.tsx` file contains a hook called `useMeta` which the `SettingsLayout` uses to populate the headers on settings pages. 

Ex.
![image](https://user-images.githubusercontent.com/65426560/196803327-37e8982b-a5b1-414f-ad69-3ac35f47d9b8.png)

I was thinking that since we're adding a CTA in the header it would be best to include it as a prop in the <Meta /> component. Zomars agrees that using the <Meta /> component is only a workaround for now and there is room for improvement.

Reply:
but why do we need this hook if we have `<HeadSEO>` that does the same, no?